src/tests/ftest/container/open.py

```diff
@@ -121,20 +118,14 @@ def test_container_open(self):
         ]
         result_messages = [messages_case_1, messages_case_2, messages_case_3]
 
-        # Create the pool and connect. Then create the container from the pool
-        # Add the pool and the container created into a list
+        # Create the pool and connect. Then create a container with the pool.
+        # Add the pool and the container created into the list.
         container_uuids = []
         for _ in range(2):
-            self.pool.append(TestPool(self.context, self.get_dmg_command()))
-            self.pool[-1].get_params(self)
-            self.pool[-1].create()
-            self.pool[-1].connect()
-            self.container.append(
-                TestContainer(pool=self.pool[-1],
-                              daos_command=DaosCommand(self.bin)))
-            self.container[-1].get_params(self)
-            self.container[-1].create()
+            self.pool.append(self.get_pool())
+            self.container.append(self.get_container(pool=self.pool[-1]))
             container_uuids.append(uuid.UUID(self.container[-1].uuid))
+            self.pool[-1].connect()
```

Comment:
Have you considered adding a `self.pool.connect()` to the [`TestContainer.open()`](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/test_utils_container.py#L449) method? It seems like that would be a more general solution than fixing this one test.

Reply:
Done